src/pages/AppDashboard.vue

```diff
@@ -238,14 +241,21 @@ function goToConfigure() {
 const SmartContractIcon = ref(CopyIcon)
 const smartContractTooltip = ref('Click to copy')
 
-function copyAppId() {
-  SmartContractIcon.value = CheckIcon
-  smartContractTooltip.value = 'Copied'
-  copyToClipboard(appId.value)
-  setTimeout(() => {
-    SmartContractIcon.value = CopyIcon
-    smartContractTooltip.value = 'Click to copy'
-  }, 3000)
+async function copyAppId() {
+  try {
+    SmartContractIcon.value = CheckIcon
+    smartContractTooltip.value = 'Copied'
+    await copyToClipboard(appId.value)
+    toast.success('App ID copied')
+  } catch (e) {
+    console.error(e)
```

Comment:
remove `logging` if not needed

Reply:
https://github.com/arcana-network/developer-dashboard/pull/103#discussion_r901689626